server.js

```diff
@@ -21,3 +22,12 @@ app.listen(PORT, function(){
 //     headers: {Authorization: `token ${process.env.GITHUB_TOKEN}`}
 //   }))(request, response);
 // }
+function proxyGitHub(request, response) {
```

Comment:
This example doesn't use Handlebars. To use Handlebars with this code, refer to lab!